src/Controls/tests/TestCases.HostApp/Issues/Issue25468.xaml.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Change this file name from Issue25468.xaml.xaml to Issue25468.xaml after uploading screenshots